common/src/main/scala/weco/concepts/common/model/UsedConcept.scala

```diff
@@ -18,3 +20,33 @@ case class UsedConcept(
   override def toString: String =
     s"\n$canonicalId\t${identifier.toString.padTo(70, ' ')}$label"
 }
+
+object UsedConcept {
```

Comment:
It's irrelevant to this patch, but I wonder if CatalogueConcept might be a better name for this? idk, just a thought

Reply:
yes, I think I prefer that too - the naming lives in a few places (like index names) so I think I'll leave actually changing it until a point (soon) where it makes sense to blat and reindex everything.